lib/node_modules/@stdlib/math/base/assert/is-oddf/docs/repl.txt

```diff
@@ -0,0 +1,33 @@
+
+{{alias}}( x )
+    Tests if a finite single-precision floating-point 
+    number is an odd number.
```

Comment:
We can place this whole sentence in a single line, as it would be less than 80 characters.